tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1916,6 +1916,211 @@ Non-finite numbers can be suppressed using Filter(), e.g.:
+auto histo = df.Filter("rdfentry_ % 10 == 0").Histo1D<int>("fNtrack");
+histo->Draw();
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// object selection: for each event, fill histogram with array of selected pts
+tree->Draw('Muon_pt', 'Muon_pt > 100')
+~~~
+   </td>
+   <td>
+~~~{cpp}
+// with RDF, arrays are read as ROOT::VecOps::RVec objects
+df.Define("good_pt", "Muon_pt[Muon_pt > 100]").Histo1D<float>("good_pt");
+~~~
+   </td>
+</tr>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Draw the histogram and fill hnew with it
+tree->Draw("sqrt(x)>>hnew","y>0"); 
+
+// Retrieve hnew from the current directory
+TH1F *hnew = (TH1F*)gDirectory->Get("hnew");
+
+// Retrieve hnew from the current Pad
+TH1F *hnew = (TH1F*)gPad->GetPrimitive("hnew");
+~~~
+   </td>
+   <td>
+~~~{cpp}
+// Save the histogram to a file
+TFile f("myhistos.root","new");
+auto hist = df.Filter("y>0").Histo1D<float>({"myhisto","myhisto",10, 0, 10},"x");
+f.WriteTObject(hist.GetPtr());
+
+// Retrieve the histogram
+TFile f("myhistos.root");
+TH1D *h = (TH1D*)f.Get("myhisto");
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Draw a 1D Profile histogram instead of TH2F
+tree->Draw("y:x","","prof");
+
+// Draw a 2D Profile histogram instead of TH3F
+tree->Draw("z:y:x","","prof");
+~~~
+   </td>
+   <td>
+~~~{cpp}
+// Draw a 1D Profile histogram
+auto profile1D = df.Profile1D<int, float>({"profName", "profTitle", 10, 0, 10}, "x", "y");
+
+// Draw a 2D Profile histogram
+auto profile2D = df.Profile2D<int, float, double>({"profName", "profTitle", 10u, 0, 10, 10, 0, 10}, "x", "y", "z");
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// This command draws 2 entries starting with entry 5
+tree->Draw("x", "","", 2, 5);
+~~~
+   </td>
+   <td>
+~~~{cpp}
+// Range function with arguments begin, end
+auto histo_range = df.Range(5,7).Histo1D<int>("x");
+histo_range->Draw();
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Draw the X() component of the 
+// ROOT::Math::DisplacementVector3D in vec_list
+tree->Draw("vec_list.X()");
+~~~
+   </td>
+   <td>
+~~~{cpp}
+auto getX = [](vector< ROOT::Math::DisplacementVector3D> &v){RVec<double> out; for( auto p: v) out.push_back(p.X()); return out;};
+auto histo = df.Define("X",getX,{"vec_list"}).Histo1D<double>("x");
+histo->Draw();
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Draw 2D arrays
+tree->Draw("fMatrix");
+tree->Draw("fMatrix[ ][ ]");
+tree->Draw("fMatrix[2][2]");
+tree->Draw("fMatrix[][0]"); 
+// and so on: 
+// fMatrix[1][0], fMatrix[2][0], fMatrix[3][0], ... , fMatrix[n][0]
+
+~~~
+   </td>
+   <td>
+~~~{cpp}
+Reading multi-dimensional arrays is not supported in RDF. 
+GH issue exists. If such feature is needed, please add a comment: "https://github.com/root-project/root/issues/15068"
+~~~
+   </td>
+</tr>
+<tr>
+   <td>
+~~~{cpp}
+// Count number of entries in a specific dataset branch
+tree->Draw("pt"); 
+TH1* myH {static_cast<TH1*>(gPad->GetPrimitive(\"htemp\"))}; 
+myH->GetEntries();
```

Comment:
How can the number of entries in a specific dataset branch be any different from the number of entries in any other branch? There are only two ways I can imagine:

* This particular branch of the computation graph has seen some filtering. This option is not represented here, because there is no cut expression passed to TTree::Draw
* This particular branch `pt` is actually a collection of `pt` from various particles in the event. From this TTree::Draw string I'm also missing that, or maybe it's missing from the description.

Without a clearer explanation here I think we cannot make the RDataFrame example meaningful, what I see right now is the equivalent of
```
ROOT::RDataFrame df{};
auto count = *df.Count();
```